platform/android/MapboxGLAndroidSDK/src/cpp/asset_manager_file_source.cpp

```diff
@@ -16,8 +16,9 @@ namespace mbgl {
 
 class AssetManagerFileSource::Impl {
 public:
-    Impl(ActorRef<Impl>, AAssetManager* assetManager_, const ResourceOptions options) :
-        resourceOptions(options.clone()),
+    Impl(ActorRef<Impl>, AAssetManager* assetManager_, const ResourceOptions resourceOpts, const ClientOptions clientOpts) :
+        resourceOptions(resourceOpts.clone()),
+        clientOptions(clientOpts.clone()),
```

Comment:
I prefer the non-abbreviated variable names. Please change `resourceOpts` to `resourceOptions` and `clientOpts` to `clientOptions`.

Reply:
Fixed to `resourceOptions_` and `clientOptions_`.